src/command_modules/azure-cli-botservice/azure/cli/command_modules/botservice/_webutils.py

```diff
@@ -53,7 +53,7 @@ def deploy_arm_template(cli_ctx, resource_group_name,  # pylint: disable=too-man
 def web_client_factory(cli_ctx, **_):
     from azure.mgmt.web import WebSiteManagementClient
     mgmt_client = get_mgmt_service_client(cli_ctx, WebSiteManagementClient)
-    mgmt_client.web_apps.api_version='2016-08-01'
+    mgmt_client.web_apps.api_version = '2016-08-01'
```

Comment:
I don't understand the reason for this - you should use the default-api-version from the Python SDK API & that is 2018-02-01

Reply:
i think you mentioned that the app service team is planning to rev up the sdk version multiple times before ignite. Will the api-version stay the same as 2018-08-01 in them? else we ll need to re-record the test everytime that changes. 
if it's going to remain constant (for atleast some time), i will remove this line and just use the default version

Reply:
Before Ignite we will update the SDK only if needed - we will do it regularly  for new feature releases, the SDK version might not change every time, Also if you add your project & tests to be available on Visual Studio 2017 solution then , updating the tests each time is not something that appservice folks can do on CLI as well. & the version is 2018-02-01 , not 2018-08-01

Reply:
done.. i also added the relevant files to the 2017 solution